src/Randomizer.Data/Configuration/Yaml/Templates/responses.yml

```diff
@@ -273,6 +273,11 @@ LocationHasDifferentItem:
 # {0} is replaced with the name of the location
 LocationCleared:
 
+# A list of different statements that will toggle go mode on. Note that you will want to include
+# "Hey tracker," at the start, and each line is just the string (so do not include "Text: " at the
```

Comment:
Hmm, I kind of want to enforce the "Hey tracker" part. Not just to simplify the configuration but also in case we later want to customize how that part works. 

Mostly because I've wanted Tracker to have a small chance to go "Hey, call me <other name> now." and then she'll only respond to that for that session. I just never got around to doing that.